app/views/foreman_wreckingball/hosts/_status_row.html.erb

```diff
@@ -85,10 +85,9 @@
                         action_buttons(
                           display_link_if_authorized(
                             _('Remediate'),
-                            hash_for_remediate_host_path(:id => host, :status_id => host.public_send(status).id).merge(:auth_object => host, :permission => 'remediate_vmware_status_hosts'),
-                            :data => { :confirm => _('Are you sure? This might cause a service interruption.') },
-                            :method => :put,
-                            :target => :blank
+                            hash_for_schedule_remediate_host_path(:id => host, :status_id => host.public_send(status).id).merge(:auth_object => host, :permission => 'remediate_vmware_status_hosts'),
+                            :data => { :title => _('Remediate Host OS'), :submit_class => 'btn-danger' },
+                            :onclick => 'show_modal(this)'
```

Comment:
This opens the modal and then follows the link. So the modal isn't shown anymore. Either prevent the default action or use something like this: https://github.com/theforeman/foreman/blob/33d9f9eeec63da0322b7d318077c2a7e58ffff11/app/views/smart_proxies/logs/_list.html.erb#L25-L33